### PR TITLE
Updated metadata for Service Bus binding/pubsub

### PR DESCRIPTION
**Pull request by ItalyPaleAle**
See dapr/components-contrib#1783

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-bindings/servicebusqueues.md [hunk 1]
```diff
@@ -45,6 +45,8 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | `queueName` | Y | Input/Output | The Service Bus queue name. Queue names are case-insensitive and will always be forced to lowercase. | `"queuename"` |
 | `ttlInSeconds` | N | Output | Parameter to set the default message [time to live](https://docs.microsoft.com/azure/service-bus-messaging/message-expiration). If this parameter is omitted, messages will expire after 14 days. See [also](#specifying-a-ttl-per-message) | `"60"` |
 | `maxRetriableErrorsPerSec` | N | Input | Maximum number of retriable errors that are processed per second. If a message fails to be processed with a retriable error, the component adds a delay before it starts processing another message, to avoid immediately re-processing messages that have failed. Default: `10` | `10` |
+| `minConnectionRecoveryInSec` | N | Input | Minimum interval (in seconds) to wait before attempting to reconnect to Azure Service Bus in case of a connection failure. Default: `2` | `5` |
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-bindings/servicebusqueues.md:52]: For minConnectionRecoveryInSec and maxConnectionRecoveryInSec I like to also get these as values into the example YAML files (there are two in this article) and mark these as optional, so then people can see that they are choices. That way they see this copied Yaml files. Can you add these, if they are applicable in both yaml file example cases?

Comment (ItalyPaleAle) [reply daprdocs/content/en/reference/components-reference/supported-bindings/servicebusqueues.md:52]: Done

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus.md [hunk 2]
```diff
@@ -81,8 +77,8 @@ The above example uses secrets as plain strings. It is recommended to use a secr
 | `maxConcurrentHandlers` | N  |Defines the maximum number of concurrent message handlers. | `10`
 | `defaultMessageTimeToLiveInSec` | N  |Default message time to live. | `10`
 | `autoDeleteOnIdleInSec` | N  |Time in seconds to wait before auto deleting idle subscriptions. | `3600`
-| `maxReconnectionAttempts` | N  |Defines the maximum number of reconnect attempts. Default: `30` | `30`
-| `connectionRecoveryInSec` | N  |Time in seconds to wait between connection recovery attempts. Default: `2` | `2`
+| `minConnectionRecoveryInSec` | N | Minimum interval (in seconds) to wait before attempting to reconnect to Azure Service Bus in case of a connection failure. Default: `2` | `5`
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus.md:84]: Did not see the service bus file here in this PR. Same comment for including this in the YAML example here as SB queues comment above for both these new entries.

Comment (msfussell) [reply daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus.md:84]: Also is this a breaking change or will the component still accept the original names?

Comment (ItalyPaleAle) [reply daprdocs/content/en/reference/components-reference/supported-pubsub/setup-azure-servicebus.md:84]: Made the change as requested.

This is a sort-of-but-not-really breaking change. The old option is ignored (but if it's still present, we show a warning in the logs and point to the docs) as it's not relevant anymore. The behavior it applied to was not desirable anyways. See conversation in https://github.com/dapr/components-contrib/pull/1783

**Review by msfussell — APPROVED** [review]
LGTM

**Pull request merged**